docs/InstallEbpf.md

```diff
@@ -0,0 +1,26 @@
+# Installing eBPF into a Test VM
+
+Follow [VM Installation Instructions](vm-setup.md) for one-time setup of a test VM.
+Once the one-time setup has been completed, the following steps will
+install or update the eBPF installation in the VM, from a machine that
+has already built the binaries for x64/Debug or x64/Release.
+
+## Method 1
+1. Deploy the binaries to `C:\Temp` in your VM, as follows:
+    a. If you built the binaries from inside the VM, then from your ebpf-for-windows directory in the VM, do `.\scripts\deploy-ebpf -l`.  Otherwise,
+    b. If you built the binaries on the host machine, then from your ebpf-for-windows directory on the host machine, start an admin Powershell on the host machine and do `.\scripts\deploy-ebpf`.
+
+2. From within the VM, install the binaries as follows:
+    1. Start an admin command shell (cmd.exe).
+    2. Do 'cd C:\temp'.
+    3. Do 'install-ebpf.bat'.
+
+## Method 2
+Copy the build output to the host of the test VM and run the following.
+1. Create a snapshot of the test VM named **baseline**.
```

Comment:
How do you do this step?  line 19 said "run the following" implying you were going to list commands.  What's the command for this?

Reply:
Added PS command to create a checkpoint named `baseline`.